basiclti/basiclti-common/src/java/org/sakaiproject/basiclti/util/SakaiBLTIUtil.java

```diff
@@ -592,10 +594,19 @@ public static void addUserInfo(Properties ltiProps, Properties lti13subst, User
 		if (user != null) {
 			setProperty(ltiProps, BasicLTIConstants.USER_ID, user.getId());
 			setProperty(lti13subst, LTICustomVars.USER_ID, user.getId());
-			setProperty(ltiProps, BasicLTIConstants.LIS_PERSON_SOURCEDID, user.getEid());
 			setProperty(lti13subst, LTICustomVars.USER_USERNAME, user.getEid());
 			setProperty(lti13subst, LTICustomVars.PERSON_SOURCEDID, user.getEid());
 
+			CandidateDetailProvider candidateDetailProvider = ComponentManager.get(CandidateDetailProvider.class);
+			if (candidateDetailProvider != null) {
+				candidateDetailProvider.getInstitutionalNumericIdIgnoringCandidatePermissions(user, null).ifPresentOrElse(
+								id -> setProperty(ltiProps, BasicLTIConstants.LIS_PERSON_SOURCEDID, id),
+								() -> setProperty(ltiProps, BasicLTIConstants.LIS_PERSON_SOURCEDID, user.getEid())
+				);
```

Comment:
this seems pretty opinionated change. seems easier to just allow an override in per-LTI settings so that you can set the LIS_PERSON_SOURCEDID to whatever value you want e.g., lis_person_sourcedid=$user.email

Reply:
agreed I opened it as a draft PR as it is only an initial thought

Reply:
GUI would be easier for institutions but i don't want to make this into a huge change.

could also just have a hard-coded list in sakai properties of LTIs that use this. 

idk. what is easiest? could also send a new custom property if CandidateDetails is set